### DISCUSSION TITLE
How to choose molecule clusters when using the "dpgen run" command

### DISCUSSION THREAD

**Discussion by cui-hq** [deepmodeling/dpgen#1054] [Q&A]
"During the simulation, the atomic forces are evaluated by the four NN PES models, For a specific atom, if the predicted forces by these four models are consistent with each other, then the molecular cluster that centered on this atom should be found in the dataset. if the results of these four models are inconsistent with each other and the error between them is in a specific range (0.5 eV/Å < error < 1.0 eV/Å in this work), the corresponding molecular cluster will be added into the dataset. " 
It was copyed from zeng's paper puplished at nature communications in 2020, I want to know when choosing the clusters, if the   complete molecule was choosen in this process.  If I want to training a NN PES model for organic molecules in water, when choosening the clusters, if the complete organic molecule was choosen in the cluster, I don' t want to destroy the organic molecule when choosing the molecule clusters. thank you very much.

**Comment by njzjz** [comment]
This paper does not utilize DP-GEN. For the work integrated into DP-GEN, please refer _Energy & Fuels, 2021, 35 (1), 762–769_. The cluster mode can be enabled by [`use_clusters`](https://docs.deepmodeling.com/projects/dpgen/en/devel/run/param.html#argument:run_jdata[fp_style=gaussian]/use_clusters) keyword.

**Reply by cui-hq** [reply]
For example,when cutting the molecule clusters,  for a water molecule, the distance between the oxygen atom and the target atom is smaller than the cutoff while the distance between the hudrogen atom and the target atom is bigger than the cutoff, whether the whole water molecule will be allocated to the cluster or not? thank you.

**Reply by njzjz** [reply]
There are two cutoff settings, the answer for one is yes, and the other is no. See documentation for [`cluster_cutoff`](https://docs.deepmodeling.com/projects/dpgen/en/devel/run/param.html#argument:run_jdata[fp_style=gaussian]/cluster_cutoff) and [`cluster_cutoff_hard`](https://docs.deepmodeling.com/projects/dpgen/en/devel/run/param.html#argument:run_jdata[fp_style=gaussian]/cluster_cutoff_hard).
